server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_6__add_list_user_database_permission_view.sql

```diff
@@ -0,0 +1,46 @@
+--
+-- Add column `resource_type`, `resource_id` to `iam_permission` table
+--
+ALTER TABLE `iam_permission` ADD COLUMN `resource_type` VARCHAR ( 32 ) DEFAULT NULL COMMENT 'Resource type of the permission, the default value is null which means not assigned';
+ALTER TABLE `iam_permission` ADD COLUMN `resource_id` BIGINT ( 20 ) DEFAULT NULL COMMENT 'Resource ID of the permission, the default value is null which means not assigned';
+
+--
+-- Add `list_user_database_permission_view` view for query user database permissions list
+--
+create or replace view `list_user_database_permission_view` as
```

Comment:
You need to consider the performance issues of this view when the amount of data is large

Reply:
The test results show that the performance will work under private cloud and will be poorer in public cloud scenarios. The performance of the authentication system is needed, but that's something for later.